.github/workflows/notify-email.yml

```
@@ -27,7 +27,7 @@ jobs:
           # Required mail subject:
           subject: Github Actions job result
           # Required recipients' addresses:
-          to: lee.calcote@layer5.io
+          to: speak2adi@gmail.com, rudrakshpareek3601@gmail.com, lee.calcote@layer5.io
```

```suggestion
          to: ${{ github.event.inputs.to }}
```
This would allow us to specify the recepients when triggering the workflow.

---

@DelusionalOptimist could we do that later? Right now let's see how the template looks😅.